controllers/usersController.js

```diff
@@ -3,14 +3,14 @@ const { getAllUsers, getUserById } = require('../queries/usersQueries');
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
+usersController.get('/', async (req, res) => {
+  users = await getAllUsers() || [];
   res.status(200).json({ data: users });
 });
 
-usersController.get('/:id', (req, res) => {
+usersController.get('/:id', async (req, res) => {
   const { id } = req.params;
-  user = getUserById(id);
+  user = await getUserById(id);
```

Comment:
The original code you were working with should have had `const` or `let` here --- not your fault but I wanted to point it out